smartsnippets/tests/tests.py

```diff
@@ -12,6 +13,10 @@
     SmartSnippetPointer,
     Variable,
 )
+class FakeSiteAdmin(admin.ModelAdmin):
```

Comment:
Why is this needed?

Maybe add a short comment explaining why, as it's not very intuitive.